src/Domain/RegulationOrder/RegulationOrder.php

```diff
@@ -8,12 +8,11 @@
 
 class RegulationOrder
 {
-    private ?RegulationCondition $regulationCondition = null;
-
     public function __construct(
         private string $uuid,
         private string $description,
         private string $issuingAuthority,
+        private RegulationCondition $regulationCondition,
```

Comment:
Le lien avec l'ORM m'a semblé être dans le mauvais sens : `RegulationOrder.orm.xml` définit un `<one-to-one field="regulationCondition">` complet avec sa join-column, et la relation inverse était plutôt du côté RegulationCondition.

Le sens tel qu'il était avant cette PR ne permettait pas de créer une RegulationCondition sans RegulationOrder ; il aurait fallu mimer ce que fait Doctrine en manipulant l'attribut privé `$regulationCondition`.